lib/core/src/lib/form/components/widgets/index.ts

```diff
@@ -30,7 +30,7 @@ import { TextWidgetComponent } from './text/text.widget';
 import { DateTimeWidgetComponent } from './date-time/date-time.widget';
 import { JsonWidgetComponent } from './json/json.widget';
 import { BaseViewerWidgetComponent } from './base-viewer/base-viewer.widget';
-import { DisplayRichTextWidgetComponent } from './display-rich-text/display-rich-text.widget';
+import { DisplayRichTextWidgetComponent } from '../../../../../../process-services-cloud/src/lib/form/components/widgets/display-rich-text/display-rich-text.widget';
```

Comment:
Shouldn't we avoid to import into the core a widget from process-service-cloud? What do you think about it?